user_events.py

```diff
@@ -756,6 +771,9 @@ def userevent_delete(id):
             __logger.info("delete user event id: %s" % sub_event_id)
             delete_user_event(sub_event_id)
 
+    if 'superEventID' in userEvent:
```

Comment:
in line 788, there is this comment `# TODO: change this logic to access superEventID field of the subevent to delete the event.` Does it mean that the logic from 786 to 800 needs to be modified since you have added this call to `remove_subevent_from_superevent_by_eventid`?

Reply:
This has been udpated.